docs/services/motion/_index.md

```diff
@@ -291,68 +292,6 @@ moved, err := motionService.MoveSingleComponent(context.TODO(), goalPose, worldS
 {{% /tab %}}
 {{< /tabs >}}
 
-### MoveOnMap
```

Comment:
I moved this down so that the methods would align with the order shown in the table, and it makes sense comprehensively to group the endpoints that produce motion together